releaseNotes.scroll

```diff
@@ -1,8 +1,14 @@
 import header.scroll
 title Scroll Release Notes
+## A list of what has changed in Scroll releases.
+
 startColumns
 
-Here's a list of the notable changes in Scroll.
+# 71.0.0 5/06/2023
+- 🎉 added `tufteCss` beta theme.
+- 🎉 added `###`, `####` and `#####` headers.
+- ⚠️ BREAKING: slight changes to the 
```

Comment:
This change is incomplete.

Reply:
Thank you for the catch! I just pushed an update with that sentence filled out.

I'm not thrilled about version 71 but I wanted to add at least 1 more theme to illuminate and iron out the bugs and difficulties in theming in general. Theming is not great but it's at least a bit better. I think theming is not the most urgent thing but it's nice to have a better understanding of what it entails now.

I think the biggest improvements to Scroll can come from improving Grammar, making it easier to extend. Also, I started another branch to allow compilation to TeX, which I think would be fun. I think it would be nice to have all Scrolls compile not just to HTML but to nice printed TypeSet versions too.

Reply:
The Tufte theme is a great addition! I love the minimalist, clear style. The TeX output is also an important output format. 

Many years ago, I manually created a set of over 70 Beamer slides in TeX as part of a job interview process. I was complimented on the quality of the slides (I was bound determined not to use PowerPoint), but it was painful solving the TeX errors that come up regularly. 

Having a straightforward plain text input format that generates the required TeX is a big positive for Scroll. Good work, Breck (@breck7)! 👍